### PR TITLE
community[patch]: Correct the calling to collection_name in qdrant 

### PR DESCRIPTION
**Pull request by killinsun**
## Description

In #16608, the calling `collection_name` was wrong.
I made a fix for it. 
Sorry for the inconvenience!

## Issue

https://github.com/langchain-ai/langchain/issues/16962

## Dependencies

N/A

### CODE REVIEW

**Comment by sfitts** [comment]
FWIW, I logged https://github.com/langchain-ai/langchain/issues/16962 about this.

**Review by Anush008 — APPROVED** [review]
👍👍👍

**Review by Anush008 — CHANGES_REQUESTED** [review]
@killinsun, this fails `langchain/libs/community/tests/integration_tests/vectorstores/qdrant`.
As there are assertions about the metadata.

**Comment by killinsun** [comment]
@Anush008 
Thank you for your review!
I added a new commit to fix failing tests.

**Comment by killinsun** [comment]
it looks CI fails at linter from langchain/libs. I'll take a look at it.

File: libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py [hunk 1]
```diff
@@ -274,3 +281,18 @@ def test_qdrant_similarity_search_with_relevance_scores(
...
+    assert len(actual) == len(expected)
+
+    for actual_doc, expected_doc in zip(actual, expected):
+        assert actual_doc.page_content == expected_doc.page_content
+
+        assert "_id" in actual_doc.metadata
+        assert "_collection_name" in actual_doc.metadata
+
+        actual_doc.metadata.pop("_id", None)
+        actual_doc.metadata.pop("_collection_name", None)
```
Comment (KShivendu) [inline libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py:296]:
```suggestion
        actual_doc.metadata.pop("_id")
        actual_doc.metadata.pop("_collection_name")
```

Should be sufficient.

File: libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py [hunk 2]
```diff
@@ -274,3 +281,18 @@ def test_qdrant_similarity_search_with_relevance_scores(
...
+    for actual_doc, expected_doc in zip(actual, expected):
+        assert actual_doc.page_content == expected_doc.page_content
+
+        assert "_id" in actual_doc.metadata
+        assert "_collection_name" in actual_doc.metadata
+
+        actual_doc.metadata.pop("_id")
+        actual_doc.metadata.pop("_collection_name")
+
+        assert actual_doc.metadata == expected_doc.metadata
```
Comment (Anush008) [inline libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py:298]: Can be defined at `tests/integration_tests/vectorstores/qdrant/common.py` for consistency.

**Review by Anush008 — CHANGES_REQUESTED** [review]
There are more assertions at `test_add_texts.py`, `test_from_texts.py`, `test_max_marginal_relevance.py` and `async_api/` tests that are failing.

File: libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py (hunk 2 above)
Comment (killinsun) [reply libs/community/tests/integration_tests/vectorstores/qdrant/test_similarity_search.py]: Thank you. I added additional fixes for them.
I run `pytest` command for all directories in `lib/community`, but tests for qdrant are ignored for some reason, then I missed failing tests. Sorry for the inconvenience!

**Review by Anush008 — APPROVED** [review]
@killinsun, works fine now. Thank you.

**Pull request merged**